pwiz_tools/Skyline/Model/Databinding/Entities/ColumnToolTips.resx

```diff
@@ -1276,6 +1279,12 @@ using the Results Grid.</value>
   <data name="TransitionResult">
     <value>Transition Result</value>
   </data>
+  <data name="TransitionResultIsMs1">
+    <value>Whether this result was obtained from an MS1 scan.</value>
+  </data>
+  <data name="TransitionResultIsQuantitative">
+    <value>Whether this result is treated as quantitative, based on the Quantitative property of the Transition and the Full Scan Acquisition Method.</value>

```

Comment:
Explain more clearly. Maybe, "the user-modifiable Quantitative property" ... "Full Scan Acquisition Method, where MS/MS fragment ions are non-quantitative for DDA." Or something that makes it clearer why this property is needed and what really it does.